src/api/res.py

```diff
@@ -39,56 +27,22 @@ def res(
         List[Union[float, int, str, Decimal, Tuple[Union[float, int, str, Decimal], str]]],
         None,
     ] = None,
-    sigfigs: Union[int, None] = None,
-    decimal_places: Union[int, None] = None,
-) -> PrintableResult:
-    return res(name, value, uncert, "", sigfigs, decimal_places)
-
-
-@overload
-def res(
-    name: str,
-    value: Union[float, int, str, Decimal],
-    sigfigs: Union[int, None] = None,
-    decimal_places: Union[int, None] = None,
-) -> PrintableResult:
-    return res(name, value, [], "", sigfigs, decimal_places)
-
-
-@overload
-# pylint: disable=too-many-arguments
-def res(
-    name: str,
-    value: Union[float, int, str, Decimal],
-    sys: Union[float, Decimal],
-    stat: Union[float, Decimal],
     unit: str = "",
+    sys: Union[float, int, str, Decimal, None] = None,
+    stat: Union[float, int, str, Decimal, None] = None,
     sigfigs: Union[int, None] = None,
     decimal_places: Union[int, None] = None,
-) -> PrintableResult:
-    return res(name, value, [(sys, "sys"), (stat, "stat")], unit, sigfigs, decimal_places)
+):
```

Comment:
Why do you not return `PrintableResult` anymore?

Reply:
I do return it in line 94. Make sure to check out the code locally to see the new contents.
It's just the GitHub diff that shows this in a weird way.

Reply:
@paul019 Did you find the statement?